src/postgraphql/http/createPostGraphQLHttpRequestHandler.js

```diff
@@ -21,14 +23,26 @@ const httpError = require('http-errors')
 const parseUrl = require('parseurl')
 const finalHandler = require('finalhandler')
 const bodyParser = require('body-parser')
+const sendFile = require('send')
+
+const { POSTGRAPHQL_ENV } = process.env
 
 const debugGraphql = new Debugger('postgraphql:graphql')
 const debugRequest = new Debugger('postgraphql:request')
 
+const graphiqlDirectory = resolvePath(__dirname, '../graphiql/public')
+
 const favicon = new Promise((resolve, reject) => {
   readFile(resolvePath(__dirname, '../../../resources/favicon.ico'), (error, data) => {
     if (error) reject(error)
-    else resolve(data)
+    else resolve(data.toString())
+  })
+})
+
+const origGraphiqlHtml = new Promise((resolve, reject) => {
+  readFile(resolvePath(__dirname, '../graphiql/public/index.html'), (error, data) => {
+    if (error) reject(error)
+    else resolve(data.toString())
```

Comment:
This is a good point that I did not think about. Same comment on the `fs-promise` thing though.